pkg/lib/ratelimit/ratelimits.go

```diff
@@ -175,13 +175,6 @@ func (n RateLimit) resolvePerIP(cfg *config.AppConfig, featureCfg *config.Featur
 	case RateLimitMessagingEmail:
 		return resolveConfig(cfg.Messaging.RateLimits.EmailPerIP, nil, featureCfg.Messaging.RateLimits.EmailPerIP)
 
-	// OAuth
-	case RateLimitOAuthTokenGeneral:
-		return resolveConfig(&config.RateLimitConfig{
-			Enabled: func() *bool { var t = true; return &t }(),
-			Period:  "1m",
-			Burst:   120,
-		}, nil, nil)
```

Comment:
Why did we move the defintions? I remember we decided to move all definitions to here recently.

Reply:
This is because in token endpoint, the per ip rate limit and per user rate limit is not checked at the same point. Therefore, unlike other ratelimits, `ResolveBucketSpecs` cannot be used. Instead, I have to define separate buckets and use them separately. So I decided to move this specific bucket into the place which uses it.